checks/namespace.go

```diff
@@ -182,7 +181,6 @@ func (c *NamespaceChecker) Check(extConfig external.Check) *pkg.CheckResult {
 	deleteOk := true
 	deletion := NewTimer()
 	if err := pods.Delete(context.TODO(), pod.Name, metav1.DeleteOptions{}); err != nil {
-		deleteOk = false
```

Comment:
Do we have tests to validate this change?

Reply:
In the e2e test we are running all the checks including this namespace one, also if you see the line below we are returning fields from the function without any reference to this var... The var is used when returning a success scenario